web-console/3-project-details/binding-in-project.md

```diff
@@ -0,0 +1,77 @@
+
+
+# Binding in Context
+
+- Users may bind Provisioned Services to other applications in their project and vice versa.
+- The binding flow will use the [Overlay Panel Pattern](http://openshift.github.io/openshift-origin-design/web-console/4-patterns/overlay-panel) and employ a similar wizard experience to the [ordering experience](http://openshift.github.io/openshift-origin-design/web-console/5-components/order-from-catalog).
+
+
+
+
+## Initiating the bind flow
+
+![template](img/bind_kebab.png)
+- A user can initiate a binding from the kebab menus of provisioned services, or from other resources in the overview that can be bound to (anything with pods).
+
+![template](img/bind_create_unbound.png)
+- For provisioned services that do not have any bindings, there is a "Create binding" action in the collapsed state.
+
+![template](img/bind_create_expanded.png)
+- The "Create binding" action is always available for Provisioned Services in the expanded state.
+
+![template](img/bind_application_expanded.png)
+- A binding can also be initiated from a deployment config (or similar) in the expanded view IF that deployment is already bound to another service. The “Service Bindings” section does not appear in the overview if there are no bindings.
+
+
+## Bind flow
+
+### General -- MVP
+![template](img/bind_steps.png)
+  - The wizard is two steps regardless whether you start from a provisioned service or from an application:
+    1. Choose service (or application)
+    2. Confirmation
```

Comment:
I don't think service works as a generic term here--It might be a service, but it might instead be part of your application, depending on how you initiated the bind.